diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 14aa00219af673e8cb92f567c0dfca64fe8d40e8..6ff819cae1219acef643b193678849af024b58d7 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2499,6 +2499,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
@@ -8366,6 +8367,7 @@ static_library("browser_public_dependencies") {
       "//rlz:rlz_lib",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
