diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index d5e960020534fbadd9b2b12325b10fc0687406f1..2d0d26180e2df9a5a37b55b795fed2e18329326a 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -479,6 +479,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
           l10n_util::GetStringUTF16(IDS_SHORT_PRODUCT_OS_NAME)));
 #endif
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
