diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index ba6521134443667cb75f32e2e6f11202d82e57de..27c03abe5a3084e51519176c01f219d8598d064b 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4325,6 +4325,7 @@ const FeatureEntry::FeatureVariation kAudioDuckingAttenuationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
