diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index d35f7b6e65a9f4efa3a321f7df96edba633989d7..b2fbb23d67fde4052d1bf53b216513e3c5f2d328 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1451,6 +1451,7 @@ source_set("extensions") {
       allow_circular_includes_from += [ "//chrome/browser/glic:impl" ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions) {
