diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 7aea3b70dbe334ffdb0d5eff6d266e45501559a4..695cc133d82f35a14e8bec2e41a1cadbd21b1dfa 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -263,6 +263,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(DataType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
