diff --git a/components/os_crypt/sync/BUILD.gn b/components/os_crypt/sync/BUILD.gn
index 8120f75eb9d4f0b6a8921c2b0d48073c558ca334..8c294547e24c24df9406b24dda19e42b577969f4 100644
--- a/components/os_crypt/sync/BUILD.gn
+++ b/components/os_crypt/sync/BUILD.gn
@@ -88,6 +88,7 @@ component("os_crypt") {
       defines += [ "USE_KWALLET" ]
     }
   }
+  import("//brave/components/os_crypt/sources.gni") deps += brave_components_os_crypt_deps
 }
 
 static_library("test_support") {
