diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index 2a1e30ed22bc0ee219bf3cac6a24cacac2a313d0..1897df76534f7d2fbc5cce7cbc2dc77738b3c820 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -274,6 +274,7 @@ json_to_struct("prepopulated_engines") {
     ":search_engine_type",
     "//base",
   ]
+  additional_sources = brave_components_search_engines_sources
 }
 
 if (is_android) {
