diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index d745020d00eae9862fd2d5c89e7a412cdd43b7d3..aa4225e44acde0496742044f99156f5b5f9dede5 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3604,6 +3604,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
