diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 03d27eb1ff479b3bfbd189648fcdfdbe208d8ff4..8a9e724b7e7d92f976d094ff641686561af29e7c 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -796,8 +796,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
