diff --git a/BUILD.gn b/BUILD.gn
index c76a79140b32bdfedfd033d5e78da7b7e578bbea..adce7d7a0cf7db94380ed22b6507529e3939285c 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -815,6 +815,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3159,6 +3160,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
