diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 1e6976aad1282c35512e6f86f05908cdde17e903..14daa794f18019a355169c887e10c23662c33d83 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3352,6 +3352,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
